### PR TITLE
Adjust getDateString to get exact time if in 6days

### PR DESCRIPTION
**Pull request by OFFLlNE**
Before if the date was more than 48hours away, then we formatted out the exact time and just displayed the date for medium/short.
With this change we will show the time if the date is within 6days (144 hours).

Was:
image

Will:
image

Personally do not see any downsides to that, only being more transparent with time

### CODE REVIEW

File: src/services/date/date.service.js [hunk 1]
```diff
@@ -349,10 +349,10 @@ function DateService() {
     currentYear,
     format
   ) => {
-    const fourtyEightHours = 48 * 60 * 60 * 1000;
+    const sixDaysInHours = 144 * 60 * 60 * 1000;
```
Comment (rs6g10) [inline src/services/date/date.service.js]: For better readability the protocol is already implemented in below line:
```
sixDaysInHours = 6 * 24 * 60 * 60 * 1000
```

Comment (rs6g10) [reply src/services/date/date.service.js]: Yea; `sixDays` 😃

**Comment by StevePole** [comment]
What motivated this?  We rarely know when things will happen with such precision of they are further away, that's why the time was omitted.

There is a proposal to change this formatter to allow an optional 'max-precision' parameter, here: https://transferwise.atlassian.net/wiki/spaces/TE/pages/1017054271/Date+formatting

**Comment by OFFLlNE** [comment]
@StevePole Oh nice, I did not know this has such beautiful documentation on it. Do you want me to update it as well?

But the reason why we did the change was that quite often guaranteed rate is promised to a date which is 48+ hours away. Which causes problems we display "We need to receive the money on Monday 3rd" So it gives them the idea that they have the whole day, but actually it would be something like `Monday 3rd 1PM` or something similar. And since this was our error we had to refund if people started complaining about that (We did send on 3rd as your website asked)

**Pull request merged**